packages/starknet-snap/src/utils/snapUtils.ts

```diff
@@ -680,3 +681,19 @@ export function toMap<k, v, z>(arr: Array<v>, key: string, keyConverter?: (v: z)
     return map;
   }, new Map<k, v>());
 }
+
+export function dappUrl(envt: string) {
```

Comment:
convert envt in lower case

Reply:
Done

Reply:
nic:
envt may be empty/underfined and envt.toLowerCase() may fail, but it can ignore if we ensure it will not empty